integration_tests/e2e/notifications-settings.cy.ts

```diff
@@ -43,7 +43,7 @@ context('A staff member can view their notification settings', () => {
 
     page.errorSummary().contains('Select if you want to receive notifications')
 
-    page.radio('EMAIL').click()
+    page.radioLabel('Email').click()
```

Comment:
test failing locally since radio is obscured by the label. Better normally to click the label anyway